src/client/styles/objects/columns.scss

```diff
@@ -0,0 +1,6 @@
+.columns {
+    padding: 0px;
```

Comment:
With the value '0' you don't need 'px'

Reply:
there are some stylint-checks always ensuring a unit. Thus, I always add it